### DISCUSSION TITLE
Alt-Enter in the picker does not open selected file

### DISCUSSION THREAD

**Discussion by MuratOzsoyler** [helix-editor/helix#9374] [General]
### Summary

`Alt-Enter` should open selected file while file picker remains open. But no files are opened.

### Reproduction Steps

 

I tried this:

1. `hx .`
2. Enterd `Space` then `f`
3. Selected an entry.
4. `Alt-Enter`

I expected the file opened and file picker remained open.

Instead, this happened:
File picker stayed open but no file is opened in the background.

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>
No log entries related to this bug, except may be this:
```
2024-01-17T22:36:05.154 helix_tui::backend::crossterm [DEBUG] The keyboard enhancement protocol is not supported in this terminal (checked in 1.956676ms)
```

</details>

### Platform

Windows 11 wsl archlinux

### Terminal Emulator

alacritty for Windows

### Installation Method

from AUR helix-git package

### Helix Version

helix 23.10 (3f88a3f4)

**Comment by David-Else** [comment]
It works here on the latest master, the file appears in the buffer list.

**Comment by the-mikedavis** [comment]
This is intentional actually, see https://github.com/helix-editor/helix/pull/4435

**Reply by MuratOzsoyler** [reply]
I've tested this on an archlinux installed computer. Indeed it works on Linux. 

But in wsl, on Windows based terminal apps (Alacritty for Windows, Windows Terminal etc.) `Alt-Enter` changes something visually on the upside of the console but nothing gets loaded. None of the `Alt-Enter`ed files appears in buffer list.

I've checked this just recently using freshly build helix-git AUR package. `hx --version` is `helix 23.10 (9c56afef)` right now.

**Reply by jw013** [reply]
Sounds like alacritty is [using Alt-Enter to toggle fullscreen](https://github.com/alacritty/alacritty/issues/2012). If you want helix to be able to see the key you will need to unbind it in alacritty.

**Reply by MuratOzsoyler** [reply]
@jw013 I think you've nailed the solution! I'll try this when I am near my computer.

**Reply by MuratOzsoyler** [reply]
Problem is this binding active only on Windows:
![image](https://github.com/helix-editor/helix/assets/12798143/34923069-e551-474b-b14c-6a4bd0ef5a69)

Thanks for your pointer and patience.
